CRM/Core/Config/MagicMerge.php

```diff
@@ -173,6 +173,7 @@ public static function getPropertyMap() {
       'recaptchaOptions' => array('setting'),
       'recaptchaPublicKey' => array('setting'),
       'recaptchaPrivateKey' => array('setting'),
+      'forceRecaptcha' => array('setting'),
```

Comment:
Not a big deal, but for future reference -- there's no general desire to pass all settings through `$config` / `MagicMerge`. `MagicMerge` is just a compatibility mechanism for legacy `$config` consumers. For new logic, use `Civi::settings()->get('foo');`

Reply:
Thanks for point this out, we will consider this for future updates.